### PR TITLE
View details front end

### PR DESCRIPTION
**Pull request by julian-tuazon**
wicked_sales_view_details_front_end

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -1,18 +1,32 @@
+/* eslint-disable no-console */
```
Comment (thebearingedge) [inline client/components/app.jsx]: Please do not turn this off.

File: client/components/product-details.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  getProductData() {
+    fetch(`/api/products/${this.props.details.productId}`)
+      .then(res => res.json())
+      .then(data => this.setState({ product: data }))
+      .catch(err => console.error(err));
+  }
+
+  handleClick() {
+    this.props.setView('catalog', {});
+    this.setState({ product: null });
```
Comment (thebearingedge) [inline client/components/product-details.jsx]: I don't believe this `setState` is necessary.

File: client/components/product-details.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export default class ProductDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { product: null };
+    this.getProductData = this.getProductData.bind(this);
```
Comment (thebearingedge) [inline client/components/product-details.jsx]: You don't need to bind `this.getProductData`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by julian-tuazon** [comment]
Incorporated requested changes. Thanks

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**